exasol/toolbox/nox/_artifacts.py

```diff
@@ -129,16 +141,16 @@ def copy_artifacts(session: Session) -> None:
     Copy artifacts to the current directory
     """
 
-    dir = Path(session.posargs[0])
+    artifact_dir = Path(session.posargs[0])
```

Comment:
Think it'll be fine once sonar is actively used, but as a heads-up, it finds issues from a merged in commit.